### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by gsismandnm**
Finished my homework.

### CODE REVIEW

File: javascript/javascript1/week2/candy-helper.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+addCandy("chewing-gum",30);
+addCandy("Cookie", 10);
+
+
+
+let i=0;
+while (i<boughtCandyPrices.length){
+    spendpower += boughtCandyPrices[i];
+    i++;
+}
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/candy-helper.js]: Calculate the total spend inside `canBuyMoreCandy` function, not before it.

File: javascript/javascript1/week2/flight-booking.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+//Gender is male or female and fullName can be true or false.
+      
+function getFullname(firstName, surName, gender, formalName){
+    if (gender==="male" && formalName==="true"){
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/flight-booking.js:4]: Use formalName as a boolean, not string type. So use `true`/`false` not `"true"` or `"false"`

File: javascript/javascript1/week2/student-manager.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+    if (studentName === "") {
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/student-manager.js]: Use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) to check long empty strings also like `"   "`

**Review by zuhairtaha — CHANGES_REQUESTED** [review]

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Good job 
- I recommend you to use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension to format your code. 
You can watch this [video](https://www.youtube.com/watch?v=iGWwmVmvFPk&ab_channel=SteveGriffith-Prof3ssorSt3v3) about it

File: javascript/javascript1/week2/student-manager.js (hunk 3 above)
Comment (gsismandnm) [reply javascript/javascript1/week2/student-manager.js]: Ok. I will change it..

File: javascript/javascript1/week2/candy-helper.js (hunk 1 above)
Comment (gsismandnm) [reply javascript/javascript1/week2/candy-helper.js]: Ok. I will do it.

File: javascript/javascript1/week2/flight-booking.js (hunk 2 above)
Comment (gsismandnm) [reply javascript/javascript1/week2/flight-booking.js:4]: Ok. I will try to do it.

**Pull request merged**